backend/src/main/java/com/example/backend/risk/Risk.java

```diff
@@ -0,0 +1,27 @@
+package com.example.backend.risk;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Risk {
+    @Id
+    private String id;
+    private String projectId;
+
+    private String riskName;
```

Comment:
nur als Vorschlag: 
Du bist hier ja schon im Risk modell - d.h. deine properties müssen das eigentlich nicht nochmal wiederholen - `risk.getName()` sagt ja genau das aus. `risk.getRiskName()` braucht man eigentlich nicht. Aber is vielelicht eine persönliche Präferenz

Reply:
Ja, ich benenne die viel lieber mit className, damit ich das explizit sehen was was ist